backend/papersfeed/utils/likes/utils.py

```diff
@@ -192,3 +195,51 @@ def remove_like_collection(args):
 
     like_counts = __get_collection_like_count([collection_id], 'collection_id')
     return {constants.LIKES: like_counts[collection_id] if collection_id in like_counts else 0}
+
+def insert_like_reply(args):
+    """Insert Like of Reply"""
+    is_parameter_exists([
+        constants.ID
+    ], args)
+
+    # Reply Id
+    reply_id = args[constants.ID]
+
+    # Request User
+    request_user = args[constants.USER]
+
+    # Check Reply Id
+    if not Reply.objects.filter(id=reply_id).exists():
+        raise ApiError(constants.NOT_EXIST_OBJECT)
+
+    # Create
+    ReplyLike.objects.create(
+        reply_id=reply_id,
+        user_id=request_user.id,
+    )
+
```

Comment:
Do you think works related with notifications for reply likes will be done by me, don’t you?

Reply:
Yes I do. I just made post/like/reply and delete/like/reply.